e2e-test/src/test/java/gov/cms/ab2d/e2etest/TestRunner.java

```diff
@@ -254,6 +254,39 @@ private void verifyJsonFromfileDownload(String fileContent, JSONArray extension)
         Assert.assertEquals(length, fileContent.getBytes().length);
     }
 
+    private boolean validFields(JSONObject jsonObject) {
+        List<String> allowedFields = Arrays.asList("identifier", "item", "meta", "patient", "billablePeriod", "diagnosis",
```

Comment:
I think these makes more sense as a Set, since we're doing lookups it's O(1) vs. doing O(n) as a List, and order doesn't seem to matter. A set can still be iterated over.

Reply:
ok